app.js

```diff
@@ -45,16 +51,24 @@ function displayPhrases() {
 
     for (const phrase of character.phrases) {
         // > create an <li> element
+        const li = document.createElement('li');
         // > set the text of the li to the phrase
+        li.textContent = phrase;
         // > append the <li> to the phrases list
+        phraseList.append(li);
     }
 }
 
+// Add some conditional logic!!!!
 addButton.addEventListener('click', () => {
     // > Get the phrase from the value of the phrase input
+   if (phraseInput.value !== ' ') { 
+    // alert('Input Required')
```

Comment:
@artist0504 you need to close the `if` block:

```js
}
```

Reply:
Try not to push commits without checking to make sure they didn't break anything